src/docfx/lib/log/Telemetry.cs

```diff
@@ -63,8 +64,9 @@ public static void TrackException(Exception ex)
 
         public static void Flush()
         {
-            // Default timeout of 100 sec is used
-            s_telemetryClient.Flush();
+            // Default timeout of TelemetryClient.Flush is 100 seconds,
+            // but we only want to wait for 2 seconds at most.
+            Task.WaitAny(Task.Run(s_telemetryClient.Flush), Task.Delay(2000));
```

Comment:
do we need to swallow exceptions here?

Reply:
It is swallowed and printed at `Program.cs`